index.html

```diff
@@ -8,21 +8,74 @@
     <!--CSS Stylesheet-->
     <link rel="stylesheet" type="text/css" href="styles.css" />
     <!--Boostrap-->
-    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet"
-        integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD" crossorigin="anonymous">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD" crossorigin="anonymous">
 </head>
 
 <body class="body">
+    <!--Header-->
     <header>
         <div class="header">
+            <!--Nav-->
             <nav>
-                
+                <div class="toolbar">
+                    <div class="logo">
+                        <a href="#">
+                            <span class="my-logo">
+                                My Logo
+                            </span>
+                        </a>
+                    </div>
```

Comment:
- [ ] Kindly extract your logo from the `<nav>` section because your logo is not part of the navigation elements. your hamburger and your links should be in a` nav` tag as they are the navigation links so as to enable screen readers understand each section of your project 👍🏽

Reply:
## Hi, @vickymarz  

I have already made the required changes, but I would like to ask for your help.
I understood why to remove the `nav` logo, could you help me by giving me a quick check on that part, please.

**Good luck bro**